nibRealUI/Modules/MinimapAdv.lua

```diff
@@ -1562,7 +1562,7 @@ local function CreateFrames()
                     return active
                 end,
                 func = function(self, arg1, arg2, isChecked)
-                    _G.SetTracking(id, isChecked)
+                    C_Minimap.SetTracking(id, isChecked)
```

Comment:
Just add back the `_G.` at the beginning, and i'll merge.

Reply:
Updated with that change